src/ripple/app/ledger/InboundLedger.h

```diff
@@ -65,10 +65,10 @@ class InboundLedger
     // Called when another attempt is made to fetch this same ledger
     void update (std::uint32_t seq);
 
-    std::shared_ptr<Ledger> const&
+    std::shared_ptr<Ledger const>
     getLedger() const
     {
-        return mLedger;
+        return std::const_pointer_cast<Ledger const> (mLedger);
```

Comment:
Outside this PR but thought I'd mention `mLedger` can be assigned in `::takeHeader` when a message arrives while `::getLedger` is called.